README.md

```diff
@@ -129,6 +129,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Rollback a GitHub Release](https://github.com/author/action-rollback)
 - [Lock Closed Issues and Pull Requests after a Period of Inactivity](https://github.com/dessant/lock-threads)
 - [Get Commit Difference Count Between Two Branches](https://github.com/jessicalostinspace/commit-difference-action)
+- [Merge A Pull Request](https://github.com/juliangruber/merge-pull-request-action)
```

Comment:
It looks like there is a similar action in the Pull Requests section: "Automatically merge Pull Requests that are ready"

Could you provide more detail on how this action is different?

Reply:
is this sufficient? It's different in that it only merges the PR that you specify

Reply:
We're trying to limit duplications of actions to this list so I'm going to close this PR.